modules/luigi/phrases/flavor_scoop.yml

```diff
@@ -0,0 +1,20 @@
+type: phrases
+name: "legit_order"
+opts:
+- "{scoops} scoops of the {flavor} will be delicious! is that all?"
+- "Wow you are quite a picker! {scoops} scoops of the {flavor} then! Are we done?"
+- "Not a bad choice! {scoops} scoops of the {flavor} sound delicious. Are we good to go?"
+---
+type: phrases
+name: "need_flavor"
+opts:
+- "it would be helpful if you also told me what flavor you want {scoops} scoops of ..."
+- "You forgot the most important part! What  {scoops} scoops of flavor do you want?"
```

Comment:
I think there is a little bit too much space in front of `{scoops}`?! :) sorry, being picky ☺️

Reply:
Spaces are usually because of my copy paste style in my yml files, sorry about that.  :)